src/core/TrackContainer.cpp

```diff
@@ -124,6 +122,9 @@ void TrackContainer::loadSettings( const QDomElement & _this )
 		if( node.isElement() &&
 			!node.toElement().attribute( "metadata" ).toInt() )
 		{
+			QString trackName = node.toElement().hasAttribute( "name" ) ? node.toElement().attribute( "name" ) :
+															   node.firstChild().toElement().attribute( "name" );
+			pd->setLabelText( tr("Loading Track %1 (%2/Total %3)").arg( trackName ).arg( pd->value() ).arg( Engine::getSong()->getLoadingTrackCount() ) );
```

Comment:
It crashes here on command line render. If you wrap it in an `if`, something like this:

```
if( gui )
{
   pd->setLabelText( tr("Loading Track %1 (%2/Total %3)").arg(
   	trackName ).arg( pd->value() ).arg(
   	Engine::getSong()->getLoadingTrackCount() ) )
}
```

that should fix it.

Also. The line is much too long and wraps badly. Please try and use more lines.